autoparams-mockito/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'java-library'
+}
+
+dependencies {
+    api 'io.github.javaunit:autoparams:0.2.0'
+    api 'org.mockito:mockito-core:2.25.0'
```

Comment:
mockito-core 는 3.9.0 까지 나와있는데요
https://mvnrepository.com/artifact/org.mockito/mockito-core

2.25.0 으로 선택한 배경이 혹시 있을까요?

Reply:
공식 사이트에 의하면 https://site.mockito.org/

> Mockito 3.x requires Java 8, but otherwise doesn’t introduce any breaking changes compared to the 2.x series.

라고 합니다. 그래서 Mockito 2.x 버전을 사용하는 프로젝트에서 굳이 3.x로 올리지 않아도 되도록 하는 것이 목적이에요.